src/main/java/frc/team2412/robot/Hardware.java

```diff
@@ -83,6 +83,8 @@
     // climb can ids are range 50-59
     public static final int CLIMB_FIXED_MOTOR = 51;
     public static final int CLIMB_LIMIT_SWITCH = 9; // to be determined - digital I/O pins are 0-9
+    public static final int POST_CLIMB_SOLENOID_UPWARDS = 2; // TODO
+    public static final int POST_CLIMB_SOLENOID_DOWNWARDS = 3;
```

Comment:
Not sure if this has changed, but from what I heard last we'll have one `Solenoid` and one `DoubleSolenoid`.

Reply:
Just checked in with mech, seems like it will be 1 solenoid, may change to 2 depending on how they plumb it